gradle.properties

```diff
@@ -11,7 +11,7 @@ title=OpenAPI/Swagger Support
 projectDesc=Configuration to integrate Micronaut and OpenAPI/Swagger
 projectUrl=https://micronaut.io
 githubSlug=micronaut-projects/micronaut-openapi
-developers=Puneet Behl,Álvaro Sánchez-Mariscal,Iván López
+developers=Puneet Behl,�lvaro S�nchez-Mariscal,Iv�n L�pez
```

Comment:
This was being displayed as:

<img width="534" alt="image" src="https://user-images.githubusercontent.com/153880/181732935-1c14ab8f-62f1-4186-adeb-c3ce19b9901f.png">

🤷

Reply:
In fact, it is wrong in the published POMs: https://search.maven.org/artifact/io.micronaut.openapi/micronaut-openapi/4.4.2/jar

Reply:
I blame you for having a tilde in your name